### PR TITLE
Fix types when combining measurements

### PR DESCRIPTION
**Pull request by mosche**
Tiny fix to correct types when combining measurements. Currently the resulting type is always a Double, which is annoying during testing.

### CODE REVIEW

File: test/framework/src/main/java/org/elasticsearch/telemetry/Measurement.java [hunk 1]
```diff
@@ -54,7 +54,9 @@ public static List<Measurement> combine(List<Measurement> measurements) {
             }
             byAttr.compute(
                 m.attributes,
-                (k, v) -> (v == null) ? m.value : isDouble ? v.doubleValue() + m.getDouble() : v.longValue() + m.getLong()
+                (k, v) -> (v == null) ? m.value
```
Comment (rjernst) [inline test/framework/src/main/java/org/elasticsearch/telemetry/Measurement.java:57]: Can you turn this into a multi line lambda, so a normal if/else can be used? More than one level of ternary is incredibly difficult to visualize.

**Review by rjernst — APPROVED** [review]
LGTM, just one (already existing) nit.

File: test/framework/src/main/java/org/elasticsearch/telemetry/Measurement.java (hunk 1 above)
Comment (mosche) [reply test/framework/src/main/java/org/elasticsearch/telemetry/Measurement.java]: Sure, absolutely right 👍

**Comment by mosche** [comment]
@elasticmachine update branch

**Pull request merged**